.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [ 2.7, 2.6, 2.5 ]
+        ruby: [ '3.0', '2.7', '2.6', '2.5' ]
```

Comment:
Any reason these are quoted, as opposed to numerical forms? The diff looks a little weird.

Reply:
I think only `3.0` needs quoting as per the comment in the example at https://github.com/ruby/setup-ruby/#matrix-of-ruby-versions referencing https://github.com/actions/runner/issues/849

I'd prefer it if we just quoted `3.0` and put a comment referencing the issue like in the example.

Reply:
Hold up. How (or why) is the difference between an integral and a decimal numeral significant?

Reply:
🤷 From the comments in the issue, it looks to be something the YAML parsing C# code is doing.